src/ol_infrastructure/applications/bootcamp/__main__.py

```diff
@@ -0,0 +1,113 @@
+import pulumi
+import pulumi_aws as aws
+
+bucket = aws.s3.Bucket(
+    "bucket",
+    arn="arn:aws:s3:::bootcamp-static-assets-test",
+    bucket="bootcamp-static-assets-test",
+    hosted_zone_id="Z3AQBSTGFYJSTF",
+    request_payer="BucketOwner",
+    tags={
+        "OU": "bootcamps",
+        "business_unit": "bootcamps",
+    },
+    opts=pulumi.ResourceOptions(protect=True),
+)
+
+bucket_app_ci = aws.s3.Bucket(
+    "bucket_app_ci",
+    arn="arn:aws:s3:::ol-bootcamps-app-ci",
+    bucket="ol-bootcamps-app-ci",
+    hosted_zone_id="Z3AQBSTGFYJSTF",
+    request_payer="BucketOwner",
+    tags={
+        "Department": "bootcamps",
+        "Environment": "ci",
+        "OU": "bootcamps",
+        "business_unit": "bootcamps",
+    },
+    versioning=aws.s3.BucketVersioningArgs(
+        enabled=True,
+    ),
+    opts=pulumi.ResourceOptions(protect=True),
+)
+
+ol_bootcamps_app_production = aws.s3.Bucket(
+    "ol_bootcamps_app_production",
+    arn="arn:aws:s3:::ol-bootcamps-app-production",
+    bucket="ol-bootcamps-app-production",
+    hosted_zone_id="Z3AQBSTGFYJSTF",
+    request_payer="BucketOwner",
+    tags={
+        "Department": "bootcamps",
+        "Environment": "production",
+        "OU": "bootcamps",
+        "business_unit": "bootcamps",
+    },
+    versioning=aws.s3.BucketVersioningArgs(
+        enabled=True,
+    ),
+    opts=pulumi.ResourceOptions(protect=True),
+)
+
+ol_bootcamps_app_rc = aws.s3.Bucket(
+    "ol_bootcamps_app_rc",
+    arn="arn:aws:s3:::ol-bootcamps-app-rc",
+    bucket="ol-bootcamps-app-rc",
+    hosted_zone_id="Z3AQBSTGFYJSTF",
+    request_payer="BucketOwner",
+    tags={
+        "Department": "bootcamps",
+        "Environment": "rc",
+        "OU": "bootcamps",
+        "business_unit": "bootcamps",
+    },
+    versioning=aws.s3.BucketVersioningArgs(
+        enabled=True,
+    ),
+    opts=pulumi.ResourceOptions(protect=True),
+)
+
+
+bootcamp = aws.rds.Instance(
```

Comment:
This should be using the `OLAmazonDB` component resource that we are using everywhere else so that it uses the proper configuration that we are supporting across our RDS instances.

Reply:
:+1: will add that...did not know that we had a component. Could you add that here please...

Reply:
Can you add that in the  issue or post a link here. I did not know that we had this since this is my first time working on a RDS instance.

Reply:
The RDS component resource is defined [here](https://github.com/mitodl/ol-infrastructure/blob/main/src/ol_infrastructure/components/aws/database.py) and you can see an example of its use [here](https://github.com/mitodl/ol-infrastructure/blob/main/src/ol_infrastructure/applications/ocw_studio/__main__.py)